kernel-switcher/switch_kernel.py

```diff
@@ -140,12 +140,55 @@ def parse_args(argv):
         help="do not write anything to the grub config",
     )
 
-    args = parser.parse_args(argv[1:])
-    return args.kernel[0], args.dry_run
+    parser.add_argument(
+        "--enable-efi-vars",
+        action="store_true",
+        help="enable EFI variables so MAAS can reprovision the machine",
+    )
+
+    return parser.parse_args(argv[1:])
```

Comment:
I don't think the argv[1:] is actually required, I've used this before to make my parse_args() function easier to test, and it always seems fine to just pass argv to it. Also useful for avoiding typeerror on the off chance that argv==None (which is technically the default but unlikely, yes).